server/player/player.go

```diff
@@ -1661,10 +1667,6 @@ func (p *Player) AttackEntity(e world.Entity) bool {
 	p.SwingArm()
 
 	i, _ := p.HeldItems()
-	living, ok := e.(entity.Living)
-	if !ok {
-		return false
-	}
```

Comment:
Bring back this if statement to keep the old functionality so non-living entities don't get past this point. Don't need the first line though since we already have that check. Might be better to rename the ok to something like isLiving though just so it's more clear

Reply:
The way you've now got this code means the attack event will not be called for non-living entities which is not what the behaviour was before. That's why I'm saying to change it

Reply:
> The way you've now got this code means the attack event will not be called for non-living entities which is not what the behaviour was before. That's why I'm saying to change it

got it